docs/source/tut-basic.rst

```diff
@@ -0,0 +1,18 @@
+.. _tutorials-basic-chapter:
+
+#################
+Tutorials - Basic
+#################
+
+.. note::
+
+   Being developed!
```

Comment:
Being developed! --> Coming soon!

(this should be changed everywhere I think)